docs: added alpha jetbrains gateway coder plugin | remove code-server version parameter (#5900)
coder/coder: docs/ides/gateway.md

# JetBrains Gateway

> JetBrains Gateway is a compact desktop app that allows you to work remotely with a JetBrains IDE without even downloading one. [See JetBrains' website to learn about and Gateway.](https://www.jetbrains.com/remote-development/gateway/)

Reviewer: ```suggestion
JetBrains Gateway is a compact desktop app that allows you to work remotely with a JetBrains IDE without even downloading one. [Learn more about JetBrains Gateway](https://www.jetbrains.com/remote-development/gateway/)
```

1. Open Gateway and click the gear icon at the bottom left and then "Settings"
1. In the Marketplace tab within Plugins, type Coder and then click "Install" and "OK"
   ![Gateway Settings and Marketplace](../images/gateway/plugin-settings-marketplace.png)
1. Click the new "Coder" icon on the Gateway home screen
   ![Gateway Connect to Coder](../images/gateway/plugin-connect-to-coder.png)
1. Enter your Coder deployment's Access Url and click "Connect" then paste the Session Token and click "OK"
   ![Gateway Session Token](../images/gateway/plugin-session-token.png)
1. Click the "+" icon to open a browser and go to the templates page in your Coder deployment to create a workspace
1. If a workspace already exists but is stopped, click the green arrow to start the workspace
1. Once the workspace status says Running, click "Select IDE and Project"
   ![Gateway IDE List](../images/gateway/plugin-select-ide.png)
1. Select the JetBrains IDE for your project and the project directory then click "Start IDE and connect"
   ![Gateway Select IDE](../images/gateway/plugin-ide-list.png)
   ![Gateway IDE Opened](../images/gateway/gateway-intellij-opened.png)

> > Note the JetBrains IDE is remotely installed into `~/. cache/JetBrains/RemoteDev/dist`

Reviewer: Small extra space

```suggestion
> Note the JetBrains IDE is remotely installed into `~/.cache/JetBrains/RemoteDev/dist`
```
